orttraining/orttraining/python/training/onnxblock/engine/training_optimizer.py

```diff
@@ -0,0 +1,54 @@
+from onnxruntime.capi import _pybind_state as C
+
+
+class TrainingOptimizer:
+    """
+    Class for running Optimize Step in Training.
+    This class is a wrapper of Optimizer Class.
+    """
+
+    def __init__(self, train_optimizer_uri, model, env=None, session_options=None, providers=None, **kwargs) -> None:
```

Comment:
we can keep the constructor simple and just keep it to a bare minimum. We can remove env, session_options, providers, kwargs